var/spack/repos/builtin/packages/py-avro-python3/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAvroPython3(PythonPackage):
+    """Apache Avro™ is a data serialization system."""
```

Comment:
Am I correct that this package is going to be deprecated? It is fine to add to Spack either way, but if it is going to be deprecated that might be worth mentioning in the Package docstring.